internal/provider/resource_wlan.go

```diff
@@ -343,6 +383,18 @@ func resourceWLANSetResourceData(resp *unifi.WLAN, d *schema.ResourceData, meta
 	d.Set("wlan_band", resp.WLANBand)
 	d.Set("no2ghz_oui", resp.No2GhzOui)
 
+	if resp.MinrateNgEnabled {
+		if resp.MinrateNgDataRateKbps == 5500 {
+			d.Set("minimum_data_rate_2g", "5.5Mbps")
+		} else {
+			d.Set("minimum_data_rate_2g", fmt.Sprintf("%dMbps", resp.MinrateNgDataRateKbps/1000))
```

Comment:
If the API is passing this as a float then I think that further supports the use of `schema.TypeFloat` for these attributes